demos/common/include/samples/common.hpp

```diff
@@ -50,6 +50,22 @@ static UNUSED std::string fileNameNoExt(const std::string &filepath) {
     return filepath.substr(0, pos);
 }
 
+static UNUSED std::ostream &operator<<(std::ostream &os, const InferenceEngine::Version *version) {
+    os << "\n\tAPI version ............ ";
+    if (nullptr == version) {
+        os << "UNKNOWN";
+    } else {
+        os << version->apiVersion.major << "." << version->apiVersion.minor;
+        if (nullptr != version->buildNumber) {
+            os << "\n\t" << "Build .................. " << version->buildNumber;
+        }
+        if (nullptr != version->description) {
+            os << "\n\t" << "Description ....... " << version->description;
+        }
+    }
+    return os;
+}
+
```

Comment:
There already is an output operator for IE versions, directly below this. You just need to update the demo code to dereference the version when printing it.

Reply:
Perhaps you're right, but it will lead to some difference between OpenVINO IE samples and OMZ demos. Anyway, I think, if OMZ contains some code copied from IE samples/common, there are no limitations to extend the copied part

Reply:
> Perhaps you're right, but it will lead to some difference between OpenVINO IE samples and OMZ demos.

It doesn't matter. There's already plenty of differences.

> Anyway, I think, if OMZ contains some code copied from IE samples/common, there are no limitations to extend the copied part

It doesn't work like that. You can't just add code because it's in IE; it has to be good.